release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -40,6 +40,41 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-7-rhcos"]
 === {op-system-first}
 
+[id="ocp-4-7-enhanced-disk-provisioning"]
+==== Enhanced disk provisioning for LUKS and RAID devices
+
+{op-system} 4.7 includes several improvements to disk provisioning for bare metal deployments:
```

Comment:
@Prashanth684 Should we mention multipath?

Reply:
yes, i think so since it is an officially supported feature ? Also need to mention that it works on clusters installed with 4.7+ only.

maybe an example can be provided on how it can be setup as a day 2 operation with a sample machine config like:
```
apiVersion: machineconfiguration.openshift.io/v1
kind: MachineConfig
metadata:
  labels:
    machineconfiguration.openshift.io/role: "master"
  name: 99-master-kargs-mpath
spec:
  kernelArguments:
    - 'rd.multipath=default'
    - 'root=/dev/disk/by-label/dm-mpath-root'
```